wks-recruiter-interface/src/app/layout/tests/test-translation/test-translation.component.ts

```diff
@@ -15,12 +15,26 @@ export class TestTranslationComponent implements OnInit {
 
   private name: string;
   private language: String;
+  private currentLanguage: String;
   private tests: Array<Test>;
+  private languages;
 
   constructor(public activeModal: NgbActiveModal,
-    private alertsService: AlertsService) { }
+    private alertsService: AlertsService) {
+  }
 
   ngOnInit() {
+    this.languages = [
+      {id: "polish", name: "polish"},
```

Comment:
Skoro id jest takie samo jak name to po co obie zmienne? Nie można tylko listy stringów zrobić?